### PR TITLE
Add 'utf8' package

### PR DESCRIPTION
**Pull request by rbsheth**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/rbsheth/hunter/builds/21651423
  * https://travis-ci.com/rbsheth/hunter/builds/97483722

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/329

* My change will resolve this "New package" request: https://github.com/ruslo/hunter/issues/1704

### CODE REVIEW

File: docs/packages/pkg/utf8.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    utf8
```
Comment (ruslo) [inline docs/packages/pkg/utf8.rst]: Should be just `utf`:
```
packages/pkg/utf8.rst:11:utf:["UT", "UHF", "Ute", "UT's", "Ctrl", "GCC", "IDE", "OSX", "RPC", "SDK", "isn", "iOS", "multi"]
writing output... [100%] user-guides/regular                                    
Spelling checker messages written to /home/travis/build/ruslo/hunter/docs/_spelling/output.txt
```
See CI testing:
* https://travis-ci.org/ruslo/hunter/builds/480211380#L2508-L2510

Comment (rbsheth) [reply docs/packages/pkg/utf8.rst]: fixed (I think)

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/21973065
- https://travis-ci.org/ingenue/hunter/builds/485972612?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
- https://ci.appveyor.com/project/ingenue/hunter/builds/21975217
- https://travis-ci.org/ingenue/hunter/builds/486007909?utm_source=github_status&utm_medium=notification

**Pull request merged**